static/src/js/components/AccessMethod/Select.css

```diff
@@ -0,0 +1,101 @@
+@import '@radix-ui/colors/black-alpha.css';
+
+/* reset */
+button {
```

Comment:
So in the select we are trying to unset all the styling for buttons in this component but, I'm not sure why? We should be able to accomplish this with `classNames`

Reply:
This was a remnant of the sample CSS from the Radix component's docs page; I removed it and some other unused styles when converting the CSS to SCSS.